src/shared/components/TrackIcon/style.scss

```diff
@@ -28,17 +28,17 @@ $track-radius-4: $corner-radius * 2;
 
     &.design,
     &.generic {
-      background: $tc-light-blue;
+      background: $tc-light-blue-110;
```

Comment:
@r0hit-gupta This doesn't pass WCAG level AA 

If we use tc-dark-blue-110 it will pass